filters/filter_19_Annoyances_Popups/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 19,
   "name": "AdGuard Popups filter",
-  "description": "Blocks all kinds of pop-ups that are not necessary for websites' operation.",
+  "description": "Blocks all kinds of pop-ups that are not necessary for websites' operation according to our Filter policy.",
```

Comment:
I guess the same should be changed in the `locales - en - filters.json - filter.19.description`.